lib/authc/StormpathAssertionAuthenticator.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var njwt = require('njwt');
+var AssertionAuthenticationResult = require('./AssertionAuthenticationResult');
+
+function StormpathAssertionAuthenticator(dataStore) {
```

Comment:
It's not very nice to have to send in the `DataStore` here. It should accept an `Application`  instead, and then you can get the datastore by `application.dataStore`.